data.sql

```diff
@@ -70,3 +70,37 @@ UPDATE animals SET owner_id=3 WHERE name IN ('Devimon','Plantmon');
 UPDATE animals SET owner_id=4 WHERE name IN ('Charmander','Squirtle','Blossom');
 --Dean Winchester owns Angemon and Boarmon.
 UPDATE animals SET owner_id=5 WHERE name IN ('Angemon','Boarmon');
+
```

Comment:
Please add a project title and project description to README. It looks more professional if it has a project title, description, and set of instructions to run the project.

Reply:
done